testing/src/test/java/org/motechproject/nms/testing/it/api/CallDetailsControllerBundleIT.java

```diff
@@ -883,13 +884,14 @@ public void testCallDetailsInvalidCallId() throws IOException, InterruptedExcept
     /**
      * To verify that Save Call Details API if rejected when mandatory parameter
      * "CallId" is having invalid value i.e. "CallId" is having value greater
-     * than 15 digit
+     * than 15 digit.
+     * This is a silly test since testCallDetailsInvalidCallId already covers this. sigh.
```

Comment:
If you were going to edit the comment couldn't you have at least changed 15 -> 25?